cli/dcoscli/main.py

```diff
@@ -89,43 +91,57 @@ def _main():
     subproc = Popen([executable,  command] + args['<args>'],
                     stderr=PIPE)
 
+    if log_level:
```

Comment:
if they specify --log-level we unset the env variable so it doesn't persist logging. We only persist if they explicitly set the env variable